Islandora/islandora_solution_pack_pdf
# ISLANDORA-1824

**JIRA Ticket**: (link)
[https://jira.duraspace.org/projects/ISLANDORA/issues/ISLANDORA-1824](https://jira.duraspace.org/projects/ISLANDORA/issues/ISLANDORA-1824)

* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)
[Islandora Metadata Interest Group Notes on adding dateIssued](https://github.com/islandora-interest-groups/Islandora-Metadata-Interest-Group/blob/master/Meetings/12_12_2016.md)

# What does this Pull Request do?
The islandora_pdf_form_mods.xml includes the dateIssued field. This form would replace the existing default MODS XML form that doesn't have a dateIssued.

A brief description of what the intended result of the PR will be and/or what problem it solves.
All other default MODS xml forms have a dateIssued with the exception of the PDF and large image solution packs. This change will make the PDF default MODS xml form more consistent with other forms. It will also allow for more options beyond just dateCreated for users.

# What's new?
The only change was the addition of dateIssued as a textfield in the default MODS xml form.

# How should this be tested?
This was tested locally in the XML form builder. The result from the preview was copied to Oxygen to ensure that the mods xml was valid.

A description of what steps someone could take to:
* Download this MODS xml form
* Import this form in their site and associate it to the Islandora PDF content model
* Ingest a PDF using this default form to see that the dateIssued appears


# Additional Notes:
This change doesn't require any change in documentation, any change to dependencies, modifications to other code.




# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/7-x-1-x-committers
